src/replays/replayitem.py

```diff
@@ -122,7 +122,18 @@ def update(self, message, client):
         
         self.name      = message["name"]
         self.mapname   = message["map"]
-        self.duration  = time.strftime('%H:%M:%S', time.gmtime(message["duration"]))
+        if message['end'] == 4294967295:  # = FFFF FFFF (year 2106) aka still playing
+            seconds = time.time()-message['start']
+            if seconds > 86400:  # more than 24 hours
+                self.duration = str(int(seconds/86400)) + " days<br />&nbsp;! error !"
+            elif seconds > 7200:  # more than 2 hours
+                self.duration = time.strftime('%H:%M:%S', time.gmtime(seconds)) + "<br />? error ?"
```

Comment:
It will be in the log in case it causes problems somewhere else (say, someone tries to launch the live replay and it doesn't work because it's invalid - we'd see that in the log and could check if it's related)